src/containers/oracle.js

```diff
@@ -18,7 +18,7 @@ import topicActions from '../redux/topic/actions';
 const RadioGroup = Radio.Group;
 const QTUM = 'QTUM';
 const BOT = 'BOT';
-const DEFAULT_RADIO_VALUE = 0;
+const DEFAULT_RADIO_VALUE = 1;
```

Comment:
curious why is this defaulting to 1 now?

Reply:
Good call. If 1 the first option is selected after toggle. After testing I decide to change it back to 0 where no option is selected initially.